test/integration/render.test.js

```diff
@@ -14,6 +14,6 @@ describe('Render tests:', () => {
 function test(file) {
     it(util.getName(file), () => {
         const actual = util.testSST(file, template);
-        return chai.expect(actual).to.eventually.be.true;
+        return chai.expect(actual).to.eventually.be.at.most(1);
```

Comment:
I would add here a comment saying that this is a temporary threshold to cover the differences in the renderer in Mac/Linux.